### PR TITLE
[WAS-495] - WAS AWS script upgrade

### PR DESCRIPTION
**Pull request by cenkawolfram**
* Upgraded ingress-controller **0.4.5** -> **1.2.1**
* Updated metrics-server/metrics-server-deployment
beta.kubernetes.io/os -> **kubernetes.io/os**
* Upgraded terraform 0.14 -> **1.2.4**
* Upgraded hashicorp/aws 3.0 -> **3.75.2**
* Upgraded terraform-aws-modules/vpc/aws 2.66.0 -> **3.14.2**
* Upgraded terraform-aws-modules/eks/aws 13.2.1 -> **16.1.0**
* Upgraded kubernetes cluster version 1.20 -> **1.22**
(AWS EKS supports cluster version <= v1.22 https://docs.aws.amazon.com/eks/latest/userguide/kubernetes-versions.html)
* Upgraded aws cli 1.XX -> **2.XX**
* Update setup script
* Updated README.md

### CODE REVIEW

**Review by delaix — APPROVED** [review]

File: EnvironmentSetup/AWS/README.md [hunk 1]
```diff
@@ -9,9 +9,9 @@ This document describes the setup of Amazon Kubernetes (EKS) and Wolfram Applica
 
 The following CLI tools are required to be installed on your local machine to complete the setup and installation:
```
Comment (manjuchandrika) [inline EnvironmentSetup/AWS/README.md:10]: @delaix This Kubernetes and AKS version upgrade is going to be major release? Shall we move our version to 3.0.0?

Comment (delaix) [reply EnvironmentSetup/AWS/README.md:10]: A major release means that things may not be compatible with prior releases. Are these changes that significant?

Comment (cenkawolfram) [reply EnvironmentSetup/AWS/README.md:10]: Actually yes, ingress-controller 1.2.1 is not compatible with the previous cluster version. EFS images are different.

Comment (delaix) [reply EnvironmentSetup/AWS/README.md:10]: OK, version 3 it is.

**Pull request merged**